src/test/load/swagger.js

```
@@ -0,0 +1,205 @@
+    utils.checkResponse(
+      response,
+      `${section.method} ${section.path} responds`,
+      res => res.status >= 200 && res.status < 600
+    );
+
+    sleep(0.5);
+  });
+
+  const authHeaders = Object.assign({}, sharedHttpParams, {
+    headers: Object.assign({}, sharedHttpParams.headers, {
+      Authorization: 'Bearer test-token-' + Math.floor(Math.random() * 1000),
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)